Library/Homebrew/extend/os/linux/cleanup.rb

```diff
@@ -1,23 +1,29 @@
 # typed: strict
 # frozen_string_literal: true
 
-module Homebrew
-  class Cleanup
-    undef use_system_ruby?
+module OS
+  module Linux
+    module Cleanup
```

Comment:
Do you have any thoughts about:
- putting this in a base `Homebrew` (or `HomebrewOS`) module?
- whether `extend` is in the module hierarchy anywhere?

Reply:
- We originally had this as `Homebrew::OS::…` but that's a breaking change, requiring full-path (`::OS`) resolution when using the top-level `OS` within the `Homebrew` module.
- `HomebrewOS` reads odd to me, it makes me wonder if there's a Hombrew OS 🤔 
- I like it when paths match namespaces, so I could be sold on `(Homebrew::)Extend::OS::Linux::…`, but i think that would lead to resolution errors as described above.

Reply:
@dduugg Cool, thanks for explaining.